src/Microsoft.IdentityModel.Tokens/Json/JsonSerializerPrimitives.cs

```diff
@@ -648,6 +648,33 @@ internal static string ReadString(ref Utf8JsonReader reader, string propertyName
             return reader.GetString();
         }
 
+        /// <summary>
+        /// This method is called to enable the 'sub' claim to deserialize it as a Number then returns it as a string.
+        /// </summary>
+        /// <param name="reader">the <see cref="Utf8JsonReader"/></param>
+        /// <param name="propertyName">the property name that is being read</param>
+        /// <param name="className">the type that is being deserialized</param>
```

Comment:
nit: capitalize the first character and end with a period

Reply:
Done. Thanks Xiao!